templates/infoSection.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="info">
```

Comment:
Please use the bootstrap classes for the grid (`container`, `row`, `col-`, etc). Otherwise, the component won't be aligned with other components of the page